docs/diagrams/commands/ViewProfileCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+!include ../style.puml
+
+box Logic LOGIC_COLOR_T1
+Participant ":LogicManager" as logicManager LOGIC_COLOR_T3
+Participant ":NuSchedulerParser" as nuSchedulerParser LOGIC_COLOR_T3
+Participant "p:ViewProfileCommand" as viewProfileCommand LOGIC_COLOR_T3
+Participant "r:CommandResult" as commandResult LOGIC_COLOR_T3
+endBox
+
+box Model MODEL_COLOR_T1
+Participant "model:Model" as model MODEL_COLOR_T3
+Participant "filteredProfiles:FilteredList" as filteredList MODEL_COLOR_T3
```

Comment:
Minor nit, perhaps there's no need to name `filteredProfiles`, since you're not using the name anywhere else

Reply:
Okay will remove it